worlds/oot_soh/ShopItems.py

```diff
@@ -263,3 +282,18 @@ def price_rule(bundle, p=price): return can_afford(p, bundle)
             # Parent region shouldn't matter at all here, so just add ROOT so we don't have to make a list of all scrubs and their regions.
             add_rule(location, rule_wrapper.wrap(
                 Regions.ROOT, price_rule, world))
+            
+    # Merchant Price Rules
+    if world.options.shuffle_merchants.value > 0:
+        for slot in merchants_items_location_table.keys():
+            # Bean Merchant Only
+            if world.options.shuffle_merchants.value == 1 and slot != Locations.ZR_MAGIC_BEAN_SALESMAN:
```

Comment:
```suggestion
            if world.options.shuffle_merchants == "bean_merchant_only" and slot != Locations.ZR_MAGIC_BEAN_SALESMAN:
```
Magic numbers are harder to read